lib/Runtime/Debug/DebugContext.h

```diff
@@ -37,6 +37,9 @@ namespace Js
         // The debugger is running which means that the engine is
         // running in interpreted mode.
         Debugging,
+
+        // The debugger is running but recording is suspended.
+        RecordingSuspended,
```

Comment:
@kfarnung why we didn't made the isReparsingSource change in release/1.6 or RS3?